frontend/public/index.html

```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
What the purpose of this one actually?

Reply:
The github logo is actually a font from Font Awesome. And to use the font, we need to add this.